gno.land/pkg/gnoweb/components/view_user.go

```diff
@@ -0,0 +1,87 @@
+package components
+
+import (
+	"net/url"
+	"time"
+)
+
+const UserViewType ViewType = "user-view"
+
+type UserLinkType string
+
+const (
+	UserLinkTypeGithub   UserLinkType = "github"
+	UserLinkTypeTwitter  UserLinkType = "twitter"
+	UserLinkTypeDiscord  UserLinkType = "discord"
+	UserLinkTypeTelegram UserLinkType = "telegram"
+	UserLinkTypeLink     UserLinkType = "link"
+)
+
+type UserContributionType struct {
+	Id   int
+	Name string
+}
+
+var (
+	UserContributionTypeRealm = UserContributionType{
+		Id:   1,
+		Name: "realm",
+	}
+	UserContributionTypePackage = UserContributionType{
+		Id:   2,
+		Name: "pure",
+	}
+)
```

Comment:
it should not be a struct in my opinion.

```suggestion
type UserContributionType int

const (
    UserContributionTypeRealm = iota
    UserContributionTypePackage
)

func (typ UserContributionType) String() string {
    switch typ {
    case UserContributionTypeRealm:
        return "realm"
    case UserContributionTypePackage:
        return "pure"
    }
}
```